### ISSUE TITLE
ErrorLog on Windows Azure tables produces wrong report title

### ISSUE THREAD

**Issue by atifaziz** [elmah/Elmah#236]
Originally reported on Google Code with ID 236

```
What steps will reproduce the problem?
1. Use windows azure table as storage 
2. Successfully pull back errors and step out of custom azure code defined as public
override int GetErrors(int pageIndex, int pageSize, System.Collections.IList errorEntryList)
3. Calling location of string hostName = Environment.TryGetMachineName(Context); found
in file ErrorLogPages.cs should be using the Azure Deployment name (or maybe instance/deployment).
In other words, this needs to also be part of the override class. Another option is
to hardcode this in the web.config.

What is the expected output? What do you see instead?
Expected out, regardless of dev box or live release up on azure, is that I see some
indication of where the call returned from on Azure.  


Please use labels and text to provide additional information.

```

Reported by `berry.dina` on 2011-07-11 12:57:03
